bh_core.sublime-settings

```diff
@@ -446,6 +446,18 @@
             "language_list": ["LaTeX", "LaTeX (TikZ)", "knitr (Rnw)"],
             "sub_bracket_search": "true",
             "enabled": true
+        },
+        // Pascal
+        {
+            "name": "pascal",
+            "open": "(?:(?<=^)|(?<=[\\s;]))(try|(?<=\\=\\s)class|(?<=\\=\\s)record|(?<=\\=\\s)interface|begin|unit|repeat)\\b",
```

Comment:
Now that I am looking at this again, I would collapse `class`, `record`, and `interface` into a common pattern: `(?<=\\=\\s)(?:class|record|interface)`.  This will clean things up a bit.